src/shared/extensionUtilities.ts

```diff
@@ -168,24 +167,35 @@ export function setMostRecentVersion(context: vscode.ExtensionContext): void {
 /**
  * Publishes a toast with a link to the welcome page
  */
-export async function promptWelcome(): Promise<void> {
+async function promptQuickStart(): Promise<void> {
     const view = localize(
-        'AWS.message.prompt.welcome.openPage',
+        'AWS.command.quickStart',
         'View Welcome Page'
```

Comment:
`AWS.command.quickStart` is not in sync with the nls file

Reply:
✔️